### PR TITLE
Bug 1107714 - [Gallery] When setting a wallpaper in landscape, the wallpaper will expand from its normal size r=djf

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[punamdahiya punamdahiya](https://github.com/punamdahiya) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=db195b667d6b)

**Comment by try-server-hook** [comment]
[punamdahiya punamdahiya](https://github.com/punamdahiya) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=11ba06fb8eb0)

File: apps/system/test/unit/wallpaper_manager_test.js [hunk 1]
```diff
@@ -42,10 +45,21 @@ suite('WallpaperManager', function() {
...
     };
 
-    // Create some image blobs
-    this.screenWidth = screen.width * window.devicePixelRatio;
-    this.screenHeight = screen.height * window.devicePixelRatio;
+    // How big (in device pixels) is the screen in its default orientation?
+    if (MockOrientationManager && !MockOrientationManager.isDefaultPortrait()) {
+      sw = Math.max(screen.width, screen.height);
+      sh = Math.min(screen.width, screen.height);
+    }
```
Comment (alivedise) [inline apps/system/test/unit/wallpaper_manager_test.js]: nit: 
} else {

**Comment by try-server-hook** [comment]
[punamdahiya punamdahiya](https://github.com/punamdahiya) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=22efc6f30a15)

**Pull request merged**